server/src/main/java/org/apache/druid/metadata/SQLMetadataSegmentManager.java

```diff
@@ -713,6 +1021,8 @@ public DataSegment map(int index, ResultSet r, StatementContext ctx) throws SQLE
                         }
                         catch (IOException e) {
                           log.makeAlert(e, "Failed to read segment from db.").emit();
+                          // If one entry is database is corrupted, doPoll() should continue to work overall. See
```

Comment:
nit: If one entry "is" -> "in" database ...

Reply:
Thanks, fixed